tests/Query/MysqlSubqueryTestCase.php

```diff
@@ -66,8 +67,9 @@ public function testGetLimitSubquerySupportsOrderByWithAggregateValuesAndDescKey
 
         $this->dbh->pop();
 
-        $this->assertEqual($this->dbh->pop(), 'SELECT DISTINCT e2.id, COUNT(DISTINCT a2.id) AS a2__0 FROM entity e2 LEFT JOIN album a2 ON e2.id = a2.user_id WHERE (e2.type = 0) GROUP BY e2.id ORDER BY a2__0 DESC, e2.name LIMIT 5');
+        $this->assertEqual($this->dbh->pop(), 'SELECT DISTINCT e2.id, e2.name, COUNT(DISTINCT a2.id) AS a2__0 FROM entity e2 LEFT JOIN album a2 ON e2.id = a2.user_id WHERE (e2.type = 0) GROUP BY e2.id ORDER BY a2__0 DESC, e2.name LIMIT 5');
```

Comment:
This query shows how the change in Query.php will affect SQL generated my Doctrine